<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QQUI_KDZ</class>
 <widget class="QWidget" name="QQUI_KDZ">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QQ</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>700</height>
    </rect>
   </property>
   <widget class="QWidget" name="page_1">
    <widget class="QLineEdit" name="lineEdit_QQID">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>170</y>
       <width>420</width>
       <height>70</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>请输入账号</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_QQpw">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>300</y>
       <width>420</width>
       <height>70</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>请输入密码</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_QQ">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>50</y>
       <width>200</width>
       <height>70</height>
      </rect>
     </property>
     <property name="text">
      <string>QQ</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_QQID">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>174</y>
       <width>180</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>账号</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_QQpw">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>302</y>
       <width>180</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>密码</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_L">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>510</y>
       <width>190</width>
       <height>85</height>
      </rect>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_R">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>510</y>
       <width>190</width>
       <height>85</height>
      </rect>
     </property>
     <property name="text">
      <string>注册</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Q">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>420</y>
       <width>110</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 8pt &quot;Agency FB&quot;;</string>
     </property>
     <property name="text">
      <string>快捷登录</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_F">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>420</y>
       <width>110</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 8pt &quot;Agency FB&quot;;</string>
     </property>
     <property name="text">
      <string>找回密码</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_RW">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>125</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QPushButton" name="pushButton_RQQ1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>125</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_QQIDnew">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>114</y>
       <width>120</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>账号</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_QQpwnew">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>273</y>
       <width>120</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>密码</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_QQpwagainnew">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>353</y>
       <width>120</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>确认密码</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_QQemailnew">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>433</y>
       <width>120</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>邮箱</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_QQcodenew">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>510</y>
       <width>120</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>验证码</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_QQnamenew">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>195</y>
       <width>120</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>昵称</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_register">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>10</y>
       <width>180</width>
       <height>75</height>
      </rect>
     </property>
     <property name="text">
      <string>注册账号</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_QQIDnew">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>110</y>
       <width>420</width>
       <height>60</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>请输入账号</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_QQnamenew">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>190</y>
       <width>420</width>
       <height>60</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>请输入昵称</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_QQpwnew">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>270</y>
       <width>420</width>
       <height>60</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>请输入密码</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_QQpwagainnew">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>350</y>
       <width>420</width>
       <height>60</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>请确认密码</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_QQemailnew">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>430</y>
       <width>420</width>
       <height>60</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>请输入邮箱</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_QQcodenew">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>510</y>
       <width>200</width>
       <height>55</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>请输入验证码</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_register">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>590</y>
       <width>720</width>
       <height>75</height>
      </rect>
     </property>
     <property name="text">
      <string>注 册</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_getcodenew">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>505</y>
       <width>130</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 8pt &quot;Agency FB&quot;;</string>
     </property>
     <property name="text">
      <string>获取验证码</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QPushButton" name="pushButton_RQQ2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>125</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_find">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>20</y>
       <width>180</width>
       <height>75</height>
      </rect>
     </property>
     <property name="text">
      <string>找回密码</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_QQIDfind">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>134</y>
       <width>120</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>账号</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_QQpwfind">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>224</y>
       <width>120</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>密码</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_QQpwagainfind">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>315</y>
       <width>120</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>确认密码</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_QQemailfind">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>404</y>
       <width>120</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>邮箱</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_QQcodefind">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>490</y>
       <width>120</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>验证码</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_QQIDfind">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>130</y>
       <width>420</width>
       <height>60</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>请输入账号</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_QQpwfind">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>220</y>
       <width>420</width>
       <height>60</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>请输入密码</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_QQpwagainfind">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>310</y>
       <width>420</width>
       <height>60</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>请确认密码</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_QQemailfind">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>400</y>
       <width>420</width>
       <height>60</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>请输入邮箱</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_QQcodefind">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>490</y>
       <width>200</width>
       <height>50</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>请输入验证码</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_find">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>585</y>
       <width>720</width>
       <height>75</height>
      </rect>
     </property>
     <property name="text">
      <string>找 回 密 码</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_QQgetcodefind">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>485</y>
       <width>130</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 8pt &quot;Agency FB&quot;;</string>
     </property>
     <property name="text">
      <string>获取验证码</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QPushButton" name="pushButton_RQQ3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>125</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_WC">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>340</y>
       <width>400</width>
       <height>65</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Arial&quot;;</string>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_quick">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>80</y>
       <width>350</width>
       <height>70</height>
      </rect>
     </property>
     <property name="text">
      <string>微信快捷登录</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_quicklogin">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>240</y>
       <width>500</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>检测到以下账号已在本机登录</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_QWC">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>510</y>
       <width>250</width>
       <height>85</height>
      </rect>
     </property>
     <property name="text">
      <string>微信登录</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
